src/main/java/com/exadel/discountwebapp/history/enums/OperationsEnum.java

```diff
@@ -0,0 +1,14 @@
+package com.exadel.discountwebapp.history.enums;
+
+import lombok.Getter;
+
+@Getter
+public enum OperationsEnum {
+    CHANGES("Changes");
+
+    public final String getOperation;
```

Comment:
rename field to a noun. this is a bad name for a field. "getOperation" is a  method name, not a field name

Reply:
renamed